esmvaltool/preprocessor/_time_area.py

```diff
@@ -61,6 +61,8 @@ def time_slice(cube, start_year, start_month, start_day, end_year, end_month,
             time=lambda t: (t_1 < time_units.date2num(t.point) < t_2))
 
     cube_slice = cube.extract(constraint)
+    if cube_slice is None:
+        raise ValueError("Time slice outside cube time boundaries.")
```

Comment:
Maybe it would be nice to mention the values of start and stop time requested and what is available in the cube?

Reply:
will do now